DigitalLearningSolutions.Data/Models/DelegateGroups/GroupCourse.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace DigitalLearningSolutions.Data.Models.DelegateGroups
+{
+    using System;
+
+    public class GroupCourse : BaseSearchableItem

```

Comment:
It's a bit odd we have to set it up as BaseSearchableItem on a page with no search, sort or filter.